src/api/types.ts

```diff
@@ -1,6 +1,13 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 export type RequestMethod = "GET" | "POST" | "PATCH" | "DELETE";
 
+export type APIArrayReturnTypes = IContact[] &
+  IInvoice &
```

Comment:
Is this the correct type you are returning? Because the name of the type contains the word `Array`. Maybe you meant it:

```typescript
export type APIArrayReturnTypes = IContact[] &
    IInvoice[] &
    IHistoryRecord[] &
    IPurchaseOrder[] &
    IQuote[] &
    IContact[] &
    Contact[];
```

Reply:
Yea it's array